src/main/java/mods/railcraft/common/core/Railcraft.java

```diff
@@ -210,7 +213,7 @@ public void serverCleanUp(FMLServerStoppingEvent event) {
     public void missingMapping(FMLMissingMappingsEvent event) {
         for (FMLMissingMappingsEvent.MissingMapping mapping : event.get()) {
             if (mapping.type == GameRegistry.Type.BLOCK) {
-                Block block = GameRegistry.findBlock(MOD_ID, MiscTools.cleanTag(mapping.name));
+                Block block = GameRegistry.findBlock(MOD_ID, mapping.resourceLocation.getResourcePath());
```

Comment:
This is completely wrong. I don't know what the correct solution is yet, but I know this wrong. The entire point of this remapping was to clean the garbage out of the tag. It might be necessary to remove it completely.